ApiFirst/ApiFirst/Controllers/CustomersController.cs

```diff
@@ -42,7 +41,7 @@ public IActionResult GetById(int id)
         [HttpPost]
         public IActionResult Create([FromBody] Customer model)
         {
-            var response = _customerService.Create(model);
+            var response = _customerAppServices.Create(model);
 
             return response
                 ? Created("", model.Id)
```

Comment:
Prefira fazer essas verificações por Try-Catch, passando argumentexceptions na sua service.
```suggestion
try 
{
var response = _customerAppServices.Create(model);
return Created("",model.Id)
}
catch (ValidationException ex)
{
return BadRequest(ex.Message)
}
```